pyproject.toml

```diff
@@ -180,21 +188,20 @@ run-tests-html-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov
 
 # Define which features and groups to include in different pixi (similar to conda) environments)
 [tool.pixi.environments]
-min-deps = ["dev", "test", "hdf", "hdf5-lib"] # VirtualiZarr/conftest.py using h5py, so the minimum set of dependencies for testing still includes hdf libs
+min-deps = ["dev", "parallel", "test", "hdf", "hdf5-lib"] # VirtualiZarr/conftest.py using h5py, so the minimum set of dependencies for testing still includes hdf libs
```

Comment:
This also seems like it should be in a separate PR?

Reply:
I removed this change in [`70602f2` (#615)](https://github.com/zarr-developers/VirtualiZarr/pull/615/commits/70602f22814a160b30a3c85eaf14b5910086b739)